ksp-fir-impl/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+description = "Kotlin Symbol Processor FIR implementation"
+
+val kotlinProjectPath: String? by project
+val intellijVersion: String by project
+val kotlinBaseVersion: String by project
+val junitVersion: String by project
+
+val libsForTesting by configurations.creating
+
+plugins {
+    kotlin("jvm")
+    id("org.jetbrains.intellij") version "0.6.4"
+    id("org.jetbrains.dokka") version ("1.4.32")
+}
+
+intellij {
+    version = intellijVersion
+}
+
+fun ModuleDependency.includeJars(vararg names: String) {
+    names.forEach {
+        artifact {
+            name = it
+            type = "jar"
+            extension = "jar"
+        }
+    }
+}
+
+dependencies {
+    implementation(kotlin("stdlib", kotlinBaseVersion))
+    implementation("org.jetbrains.kotlin:high-level-api-for-ide:1.6.255")
+    implementation("org.jetbrains.kotlin:kotlin-compiler:$kotlinBaseVersion")
+
+    implementation(project(":api"))
+
+    // workaround: IntelliJ doesn't resolve packed classes from included builds.
+    if (kotlinProjectPath != null) {
```

Comment:
Do you want composite build? I haven't used that mode for more than one year, and I feel that it is likely broken now.